main.js

```diff
@@ -3,14 +3,22 @@ require("babel-polyfill")
 
 window.engine = {}
 const app = new Tram({
+  defaultRoute: '/',
```

Comment:
added defaultRoute parameter, it's rare that this will actually be used (because we added a parameter to the main route... but hey, it's good to have)